api/events/types.go

```diff
@@ -17,12 +17,33 @@ import (
 )
 
 type LogMeta struct {
-	BlockID        thor.Bytes32 `json:"blockID"`
-	BlockNumber    uint32       `json:"blockNumber"`
-	BlockTimestamp uint64       `json:"blockTimestamp"`
-	TxID           thor.Bytes32 `json:"txID"`
-	TxOrigin       thor.Address `json:"txOrigin"`
-	ClauseIndex    uint32       `json:"clauseIndex"`
+	BlockID        thor.Bytes32     `json:"blockID"`
+	BlockNumber    uint32           `json:"blockNumber"`
+	BlockTimestamp uint64           `json:"blockTimestamp"`
+	TxID           thor.Bytes32     `json:"txID"`
+	TxOrigin       thor.Address     `json:"txOrigin"`
+	ClauseIndex    uint32           `json:"clauseIndex"`
+	OptionalData   *LogOptionalData `json:"optionalData,omitempty"`
```

Comment:
IMO we should probably have tx and log index inline here with `json:"<txIndex,clauseIndex>,omitempty"`, as we already have `ClauseIndex`